book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] The book itself should be sent as a parameter to create the new `rental` achieving the 'has-many' association.
- [ ] Kindly review the Link to the example in a previous lesson: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship`

### Sample
```
def add_rental(person, date)
    Rental.new(date, self, person)
```

Reply:
Needs attention!